test-harness/index.ts

```diff
@@ -43,60 +45,79 @@ describe('harness', () => {
 
     const testText = `${file}${os.EOL}${parsed.test}`;
 
-    test(testText, done => {
+    test(testText, async done => {
```

Comment:
In general mixing `async` and `done` callbacks together is not a good idea. We should either use one or the other system. In this case, let's stick with the `done` callback.

Reply:
You right, I updated it to use callback only.